pkg/manifests/manifests.go

```diff
@@ -134,6 +134,26 @@ func ClusterRoleBinding(component, subComponent string) (*rbacv1.ClusterRoleBind
 	return crb, nil
 }
 
+func RoleBinding(component, subComponent string) (*rbacv1.RoleBinding, error) {
+	if err := validateComponent(component); err != nil {
```

Comment:
shouldn't this be allowed only for scheduler plugins? (component == "sched")

Reply:
Yes. but that's what i'm doing 
```mf.RBController, err = manifests.RoleBinding(manifests.ComponentSchedulerPlugin, manifests.SubComponentSchedulerPluginController)```